kevm

```diff
@@ -53,8 +54,15 @@ run_krun() {
     local backend=$1  ; shift
     local run_file=$1 ; shift
     run_env "$run_file"
-    export K_OPTS=-Xss500m
-    krun --directory "$build_dir/$backend/" -cSCHEDULE="$cSCHEDULE" -pSCHEDULE='printf %s' -cMODE="$cMODE" -pMODE='printf %s' "$run_file" "$@"
+    if [[ "$backend" == "haskell" ]]; then
+        kast="$(mktemp)"
```

Comment:
Lets make the temporary file somewhere in the `.build` directory. Perhaps `$(mktemp "$build_dir"/haskell/kast.XXXXXX)`? It should be removed every time, but in case it isn't lets hide it.